frontend/components/Blocks/TitleBlock/TitleBlock.tsx

```diff
@@ -0,0 +1,30 @@
+import RawHtml from "../../RawHtml/RawHtml";
+
+type Props = {
+  data: {
+    type: string;
+    value: {
+      backgroundColor: string;
+      title: string;
+      size: string;
```

Comment:
TypeScript is not happy since a string is too general a type to be used as an element (it's being used as a component called `Tag` later in the component).

For example, `"hello world"` is a valid string, but it's not acceptable as an HTML element or component.

```suggestion
      size: React.ElementType;
```

(For what it's worth, `size` seems an odd name for this parameter. Why would `size` be an element type?)